config/src/config/inspection_service_config.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::utils;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct InspectionServiceConfig {
```

Comment:
Why not Metrics?

Reply:
This is already more than just metrics (e.g., system info, build info and node configuration) and I think we'll likely want to extend it further in the future (e.g., interactive debugging, deep introspection etc.).

Reply:
Maybe debug was a better word lol ... NodeStatus / NodeState / Metrics!